src/runtime/entitlements-manager.js

```diff
@@ -76,6 +77,7 @@ export class EntitlementsManager {
         this.positiveRetries_, opt_expectPositive ? 3 : 0);
     if (opt_expectPositive) {
       this.storage_.remove(ENTS_STORAGE_KEY);
+      this.storage_.remove(IS_READY_TO_PAY_STORAGE_KEY);
```

Comment:
Please also clear it in the `clear()` method. (You may need to sync with master).

Reply:
done